sormas-ui/src/main/java/de/symeda/sormas/ui/campaign/CampaignController.java

```diff
@@ -28,6 +28,8 @@
 import de.symeda.sormas.api.campaign.CampaignReferenceDto;
```

Comment:
Please turn on save actions in your IDE and make sure that your code is formatted upon saving. This file is not formatted correctly.

Reply:
triggered the check, however no changes seen

Reply:
The wrong formattings are still in place, are you sure that your IDE is set up correctly here? (No priority, changing the other things is more important so that I can do some testing, but please make sure to check your IDE afterwards)